go/vt/vttest/local_cluster.go

```diff
@@ -483,7 +483,7 @@ func (db *LocalCluster) loadSchema(shouldRunDatabaseMigrations bool) error {
 			}
 
 			for _, dbname := range db.shardNames(kpb) {
-				if err := db.Execute(cmds, dbname); err != nil {
+				if err := db.ExecuteWithSuperReadOnly(cmds, dbname); err != nil {
```

Comment:
Aren't we simply avoiding really testing the new functionality whenever we use this?